package/MDAnalysis/analysis/rms.py

```diff
@@ -569,67 +631,126 @@ def save(self, filename=None):
     ----
     No RMSD-superposition is performed; it is assumed that the user is
     providing a trajectory where the protein of interest has been structurally
-    aligned to a reference structure.
+    aligned to a reference structure. See the Examples section below.
 
 
-    Notes
-    -----
-    The root mean square fluctuation of an atom :math:`i` is computed as the
-    time average
+    Run the analysis with :meth:`RMSF.run`, which stores the results
+    in the array :attr:`RMSF.rmsf`.
 
-    .. math::
+    """
+    def __init__(self, atomgroup, **kwargs):
+        r"""Parameters
+        ----------
+        atomgroup : AtomGroup
+            Atoms for which RMSF is calculated
+        start : int (optional)
+            starting frame, default None becomes 0.
+        stop : int (optional)
+            Frame index to stop analysis. Default: None becomes
+            n_frames. Iteration stops *before* this frame number,
+            which means that the trajectory would be read until the end.
+        step : int (optional)
+            step between frames, default None becomes 1.
+        verbose : bool (optional)
+            if ``False``, suppress all output
 
-      \rho_i = \sqrt{\left\langle (\mathbf{x}_i - \langle\mathbf{x}_i\rangle)^2 \right\rangle}
 
-    This method implements an algorithm for computing sums of squares while
-    avoiding overflows and underflows [Welford1962]_.
+        Notes
+        -----
+        The root mean square fluctuation of an atom :math:`i` is computed as the
+        time average
 
-    References
-    ----------
-    .. [Welford1962] B. P. Welford (1962). "Note on a Method for
-       Calculating Corrected Sums of Squares and Products." Technometrics
-       4(3):419-420.
-
-    .. versionadded:: 0.11.0
-    .. versionchanged:: 0.16.0
-       Flexible weighting scheme with new `weights` keyword.
-    .. deprecated:: 0.16.0
-       Instead of ``mass_weighted=True`` use new ``weights='mass'``;
-       refactored to fit with AnalysisBase API;
-       the keyword argument `quiet` is deprecated in favor of `verbose`.
+        .. math::
 
-    """
-    def __init__(self, atomgroup, weights=None, **kwargs):
+          \rho_i = \sqrt{\left\langle (\mathbf{x}_i - \langle\mathbf{x}_i\rangle)^2 \right\rangle}
+
+        No mass weighting is performed.
+
+        This method implements an algorithm for computing sums of squares while
+        avoiding overflows and underflows [Welford1962]_.
+
+
+        Examples
+        --------
+
+        In this example we calculate the residue RMSF fluctuations by analyzing
+        the C:math:`_\alpha` atoms. First we need to fit the trajectory to the
+        average structure as a reference. That requires calculating the average
+        structure first. Because we need to analyze and manipulate the same
+        trajectory multiple times, we are going to load it into memory using
+        the :module:`~MDAnalysis.coordinates.MemoryReader`. (If your trajectory
```

Comment:
It was actually the `:module:`, should have been `:mod:`. Fixed in the force-push.